pr-dhl-woocommerce/includes/REST_API/Parcel_DE/Item_Info.php

```diff
@@ -782,6 +782,38 @@ protected function get_services_schema() {
 				'default' => '',
 				'rename'  => 'additionalInsurance',
 			),
+			'bulky_goods_intl'       => array(
+				'default'  => '',
+				'rename'   => 'bulkyGoods',
```

Comment:
This field value will be overwritten by the next field, check line 817
`'rename'  => 'bulkyGoods',`